jax/experimental/sparse/ops.py

```diff
@@ -1429,6 +1442,10 @@ def from_scipy_sparse(cls, mat, *, index_dtype=None, n_dense=0, n_batch=0):
     indices = jnp.column_stack((mat.row, mat.col)).astype(index_dtype)
     return cls((data, indices), shape=mat.shape)
 
+  def _unbatch(self):
+    """Return an unbatched representation of the BCOO matrix."""
+    return self.__class__(_unbatch_bcoo(self.data, self.indices, self.shape), shape=self.shape)
```

Comment:
Any reason not to construct `BCOO(...)` explicitly? I don't think we're expecting subclasses, and if anyone were to create one then this makes an unchecked assumption about its constructor.

Reply:
Good call, Done.